### ISSUE TITLE
The result of cifar cannot be reproduced

### ISSUE THREAD

**Issue by ThisisBillhe** [CQUlearningsystemgroup/LearningToBinarize#1]
Since there is no official training code for cifar10, I implement it with the same optimizer/scheduler and the hyperparameter as trainimagenet.py. I trained for 400epoches but the accuracy has never exceeded 65%. Are there any tricks? Or Can you release a training code on cifar?

Thx.
